caching-service/src/main/java/org/zowe/apiml/caching/service/redis/config/RedisConfig.java

```diff
@@ -113,13 +116,21 @@ private static NodeUriCredentials parseCredentialsFromUri(String nodeUri) {
 
         String credentials = nodeUri.substring(0, nodeUri.indexOf(AUTHENTICATION_SEPARATOR));
```

Comment:
Is this okay?

Reply:
Could you be more specific? I'm not sure what you mean

Reply:
We are trying to extract a password as a char array but in the beginning, we create the string `credentials`. But I am not sure if we can do anything about it.

Reply:
You are right, I see it similarly, this is already coming from the `nodeUri` string

Reply:
discussed in person, this could only affect redis which does not run on mainframe.